project/plugins.sbt

```diff
@@ -19,7 +19,7 @@ addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.4.2")
 addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "3.0.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.7")
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
-addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.28")
+addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.29")
```

Comment:
Is this upgrade needed for this PR? Because it crosses with https://github.com/akka/akka/pull/28442.
Let's kill https://github.com/akka/akka/pull/28442 if so.

Reply:
Yes, I needed one of the new changes (I think https://github.com/lightbend/sbt-paradox-apidoc/pull/108) for the apidoc links to work correctly.

Reply:
(merged #28442, looks like github was smart enough not to get confused by that)